jobs/job_runner.go

```diff
@@ -20,13 +20,15 @@ const (
 var OysterWorker = worker.NewSimple()
 
 var (
-	IotaWrapper = services.IotaWrapper
-	EthWrapper  = services.EthWrapper
-	PrometheusWrapper  = services.PrometheusWrapper
+	IotaWrapper       = services.IotaWrapper
+	EthWrapper        = services.EthWrapper
+	PrometheusWrapper = services.PrometheusWrapper
 )
 
 func init() {
-	if enabled, err := strconv.ParseBool(os.Getenv("JOB_RUNNER")); err == nil && !enabled {
+	enabled, err := strconv.ParseBool(os.Getenv("JOB_RUNNER"))
```

Comment:
Since you already have `IsInUnitTest`, can we just use that? I don't think we'd ever disable a runner outside of tests.

Reply:
Sure.